website/thaliawebsite/settings.py

```diff
@@ -682,11 +682,10 @@ def from_env(
 PHOTO_UPLOAD_SIZE = 2560, 1440
 
 # TinyMCE config
-TINYMCE_JS_URL = "/static/tinymce/js/tinymce/tinymce.min.js"
+TINYMCE_JS_URL = "/static/tinymce/tinymce.min.js"
 
 TINYMCE_DEFAULT_CONFIG = {
     "selector": "textarea",
-    "theme": "modern",
     "plugins": "link image paste code contextmenu",
```

Comment:
I don't believe the image plugin is still working

Reply:
<img width="318" alt="Screenshot 2022-02-02 at 20 37 42" src="https://user-images.githubusercontent.com/1799914/152225175-1f692a5c-d322-4563-9d6b-ae4223b88cb0.png">
Missing the button mostly

Reply:
Before:
<img width="363" alt="image" src="https://user-images.githubusercontent.com/1576660/152225317-7a0e3dfe-b981-483d-a5c4-69f7de1d7439.png">

After:
<img width="382" alt="image" src="https://user-images.githubusercontent.com/1576660/152225357-53549575-1ccf-4c74-8fd6-c01a3993e478.png">